src/languages/markdown.js

```diff
@@ -216,6 +216,11 @@ export default function(hljs) {
     end: '$'
   };
 
+  const LINEBREAK = {
+    className: 'addition',
```

Comment:
We'd need a better, more semantic scope name... this is a weird one, so I'm not sure what to suggest.

Also I don't think we want to highlight empty space - too much of a chance for false positives in better Markdown formats that don't use this weird way of marking line breaks but where someone might have extra whitespace.  The `\` at least is less likely to suffer from false positives, so could make sense for some cases.